plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -98,12 +99,14 @@ function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $b
 		 * @param string $sizes The image sizes attribute value.
 		 * @param string $size  The image size data.
 		 */
-		$filter = static function ( $sizes, $size ) use ( $block ) {
-			$id        = $block->attributes['id'] ?? 0;
-			$alignment = $block->attributes['align'] ?? '';
-			$width     = $block->attributes['width'] ?? '';
+		$filter = static function ( $sizes, $size ) use ( $block, $parsed_block ) {
+			$id                   = $block->attributes['id'] ?? 0;
+			$alignment            = $block->attributes['align'] ?? '';
+			$width                = $block->attributes['width'] ?? '';
+			$has_parent_block     = isset( $parsed_block['parentLayout'] );
```

Comment:
The `parentLayout` added in WP 6.6. See https://github.com/WordPress/WordPress/blob/e174cbef2e3b7f1901892cb501f683c6e122231b/wp-includes/block-supports/layout.php#L926-L943.

Once we bump the version in #1682 it will fix the issue.

Reply:
In https://github.com/WordPress/performance/pull/1704/commits/5a9ce372df91190a8baa32411766164a5bdc6f48 i bump the workflow WP version to 6.6